README.md

```diff
@@ -1,15 +1,62 @@
-[![TLA+ Build Status](https://github.com/tlaplus/tlaplus/workflows/CI/badge.svg?branch=master)](https://github.com/tlaplus/tlaplus/actions?query=workflow%3ACI)
+Overview
+--------
+[![](https://github.com/tlaplus/tlaplus/workflows/CI/badge.svg?branch=master)](https://github.com/tlaplus/tlaplus/actions?query=workflow%3ACI)
 
-For more information, visit http://lamport.azurewebsites.net/tla/tla.html
+This repository hosts the core TLA⁺ command line interface (CLI) tools and the Toolbox integrated development environment (IDE).
+Its development is managed by the [TLA⁺ Foundation](https://foundation.tlapl.us/).
+See http://tlapl.us for more information about TLA⁺ itself.
+For the TLA⁺ proof manager, see https://github.com/tlaplus/tlapm.
+
+Versioned releases can be found on the [Releases](https://github.com/tlaplus/tlaplus/releases) page.
+Currently, every commit to the master branch is built & uploaded to the [1.8.0 Clarke pre-release](https://github.com/tlaplus/tlaplus/releases/tag/v1.8.0).
+If you want the latest fixes & features you can use that pre-release.
+
+The CLI tools and Toolbox IDE are both written in Java.
+The Toolbox is based on the [Eclipse Platform](https://github.com/eclipse-platform/eclipse.platform).
+The CLI tool source code is primarily located in the [tlatools/org.lamport.tlatools](./tlatools/org.lamport.tlatools) directory.
+The Toolbox IDE source code is contained within the [toolbox](./toolbox) directory.
```

Comment:
A few very minor nits here:
 - since the Toolbox is on its way out, move its sentences last (order should be "both written in Java"; "CLI source code is located ..."; "Toolbox is based on Eclipse Platform and its source code is located ...")
 - cut "primarily": the tools' source code _is_ located in that folder (only the contents of that folder are necessary to build working CLI tools)
 - simplify wording: "is {located,contained} in the ./foo/bar directory" ==> "is in ./foo/bar"